### ISSUE TITLE
Unequal list of software between homebrew and non-homebrew

### ISSUE THREAD

**Issue by bjoerntorben** [MoonLiightz/dotfiles#9]
The list of software to install is different for homebrew and non-homebrew installations.

**Comment by MoonLiightz** [comment]
You're right about that. Packages to be installed via dotfiles have been added lately only for installation via Homebrew/Linuxbrew. For our Linux systems without Linuxbrew, it would be nice if the packages do not have to be installed manually. I don't see the priority for that as high, but it would definitely be nice. Thanks for creating this issue!

**Comment by MoonLiightz** [comment]
I have just noticed during a new Linux installation that the following packages are missing:
- bat (needs a `mkdir -p ~/.local/bin; ln -s /usr/bin/batcat ~/.local/bin/bat`)
- exa (then the ls alias rewrite for Mac can also take place for Linux)
